testem.json

```diff
@@ -1,6 +1,7 @@
 {
   "framework": "qunit",
-  "test_page": "tests/index.html?coverage",
+  "test_page": "tests/index.html?hidepassed",
```

Comment:
We should keep the coverage flag, but we can also add the hidepassed ?coverage&hidepassed